docker/android/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+export FASTLANE_DISABLE_COLORS=1
+export REALM_DISABLE_ANALYTICS=1
+export STATUS_RELEASE_STORE_PASSWORD=1TteVCyHykCa
+export STATUS_RELEASE_KEY_ALIAS=status-im
+export STATUS_RELEASE_KEY_PASSWORD=1TteVCyHykCa
+
+#make clean
+## Prep
+#bundle install --quiet
+#make prepare-android
+## Lint
+#lein cljfmt check
+## Test
+#lein test-cljs
+## Build
+#lein prod-build-android
+# Compile
+id
+cd android
```

Comment:
Maybe we should run `gradlew` only if `cd android` is successful?

Reply:
That's already covered by `set -e -x`.

Reply:
Besides, this was just for my own tests while making the image.